wiki/markdown/curl-examples.md

```diff
@@ -172,21 +274,64 @@ Note: It is not possible to add the `backendurl` parameter as a JSON parameter.
     </tr>
 </table>
 
-## Curl Examples
-A curl call with the command: `curl -X GET http://localhost:8080/auth` Gives the response data `{"security":false}`. The `security` parameter would be true if LDAP was enabled.
+For these endpoints to be reachable the Eiffel Intelligence back-end [needs to be configured](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/application.properties) with `testaggregated.enabled: true`.
+The below command would result in a json response of `{"status":true}` if this functionality is enabled.
+
+    curl -X GET -H "Content-type: application/json" localhost:8080/rules/rule-check/testRulePageEnabled
```

Comment:
An example with /aggregation would be more interesting since get examples is everywhere :)